staging/src/k8s.io/code-generator/cmd/conversion-gen/main.go

```diff
@@ -14,20 +14,38 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// conversion-gen is a tool for auto-generating Conversion functions.
+// conversion-gen is a tool for auto-generating functions that convert
+// between internal and external types.  A general conversion task
+// involves three packages: the one containing the internal types, the
+// one containing the external types, and the destination (i.e., the
+// one containing the conversion functions).
```

Comment:
I think "i.e, where the generated conversion functions are output to" is more clear. WDYT?

Reply:
I wrote it the way I did because that package is not only where the generated functions go but is also where the developer-authored conversion functions appear.

Reply:
Updated, to include the suggested wording and also mention the developer-authored functions.